src/passes/CoalesceLocals.cpp

```diff
@@ -438,7 +438,7 @@ void CoalesceLocals::applyIndices(std::vector<Index>& indices, Expression* root)
   }
   // update type list
   auto numParams = getFunction()->getNumParams();
-  Index newNumLocals = 0;
+  Index::value_type newNumLocals = 0;
```

Comment:
No, that's actually only there because it's passed to `std::max` and it wasn't getting implicitly converted. I put the type back and put a cast in the call to `std::max` instead.